templates/patterns/icon/icon.ui_patterns.yml

```diff
@@ -1,41 +1,40 @@
 icon:
-  label: "Icon"
-  description: "Icons available here: https://icons.getbootstrap.com/"
+  label: 'Icon'
+  description: 'A component to show an icon. For more information visit: https://icons.getbootstrap.com/.'
   settings:
```

Comment:
This is all wrong. Settings are addictional to fields. We have no fields in this pattern, it's not ok. These are all fields of an icon, except transformation which is a setting.